src/schema/v2/static_content.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  GraphQLObjectType,
+  GraphQLFieldConfig,
+  GraphQLNonNull,
+  GraphQLString,
+} from "graphql"
+import { ResolverContext } from "types/graphql"
+
+const StaticContentType = new GraphQLObjectType<any, ResolverContext>({
+  name: "StaticContent",
+  fields: {
+    name: {
+      type: GraphQLString,
+    },
+    content: {
+      type: GraphQLString,
+    },
+  },
+})
+
+const StaticContent: GraphQLFieldConfig<void, ResolverContext> = {
+  type: StaticContentType,
+  description: "Content for a specific page or view",
+  args: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The slug for the view",
```

Comment:
Is this _just_ a slug, or does it also support an `id` field?

Reply:
I talked to @ashkan18 and it and he said it could be both. Might be good to reflect that: https://artsy.slack.com/archives/CDR806U7J/p1569526745006400